k8s/bases/cse-valid-d2cc/cse-valid-d2cc-task-manager-configmap.yaml

```diff
@@ -22,6 +22,10 @@ data:
           maximumPoolSize: ${HIKARI_MAX_POOL_SIZE:10}
           minimumIdle: ${HIKARI_MIN_IDLE:5}
       jpa:
+        properties:
```

Comment:
Why didn't you perform changes for CSE Valid processes in docker-compose folder?